### DISCUSSION TITLE
Help - No accelerometer detected and auto orientation not working won't wake up from sleep

### DISCUSSION THREAD

**Discussion by sharsharbinks** [Ralim/IronOS#1006] [General]
Hey everyone,
I'm a new to soldering and decided to just go straight into ts100's. I got a ToolPAC PRO32 STM32 and just loaded the updated firmware right away. 

On bootup it then said no Accelerometer detected. The Auto orientation doesn't work (I'm left handed sometimes) and if there is any sensitivity it doesn't come out of sleep, or will go into sleep in my hands while I'm holding it. 

I bought this off eBay so I'm really hoping it's not defective and an easy fix... but kinda seems like it's a hardware issue. 

Any help or direction will be greatly appreciated. 

Edit: went back to 2.10.1 I believe that's older  https://github.com/Ralim/IronOS/releases/tag/v2.10.1
No acceleromter detected notice went away, however, still the orientation doesn't flip when using left. But i just changed it to left handed and it works fine for me.

**Comment by Ralim** [comment]
Only the later firmware builds contain the warning.
If you are seeing that warning one of two things are true:
* Accelerometer is missing or damaged
* Accelerometer has been changed to a model we don't currently have support for

Given you have bought a re-brand I can't comment to which of these is true.
I would suggest rolling back to the official firmware and confirming if it does or doesn't work there. If it doesn't work there then file a claim for replacement asap.

**Reply by sharsharbinks** [reply]
You're right! I figured out how to roll back to the official firmware and realized that it wasn't working with that either... Thank you for your help.

**Reply by Ralim** [reply]
No problem at all, happy to help
